src/CoreWCF.Primitives/src/CoreWCF/MessageSecurityVersion.cs

```diff
@@ -0,0 +1,289 @@
+        {
+            get
+            {
+                return WSSecurity11WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11BasicSecurityProfile10MessageSecurityVersion.Instance;
+            }
+        }
+
+        public static MessageSecurityVersion WSSecurity11WSTrust13WSSecureConversation13WSSecurityPolicy12
+        {
+            get
+            {
+                return WSSecurity11WSTrust13WSSecureConversation13WSSecurityPolicy12MessageSecurityVersion.Instance;
+            }
+        }
+
+        public static MessageSecurityVersion WSSecurity10WSTrust13WSSecureConversation13WSSecurityPolicy12BasicSecurityProfile10
+        {
+            get
+            {
+                return WSSecurity10WSTrust13WSSecureConversation13WSSecurityPolicy12BasicSecurityProfile10MessageSecurityVersion.Instance;
+            }
+        }
+
+        public static MessageSecurityVersion WSSecurity11WSTrust13WSSecureConversation13WSSecurityPolicy12BasicSecurityProfile10
+        {
+            get
+            {
+                return WSSecurity11WSTrust13WSSecureConversation13WSSecurityPolicy12BasicSecurityProfile10MessageSecurityVersion.Instance;
+            }
+        }
+
+        public static MessageSecurityVersion Default
+        {
+            get
+            {
+                return WSSecurity11WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11MessageSecurityVersion.Instance;
+            }
+        }
+
+        internal static MessageSecurityVersion WSSXDefault
+        {
+            get
+            {
+                return WSSecurity11WSTrust13WSSecureConversation13WSSecurityPolicy12MessageSecurityVersion.Instance;
+            }
+        }
+
+        internal MessageSecurityVersion() { }
+
+        public SecurityVersion SecurityVersion
+        {
+            get
+            {
+                return MessageSecurityTokenVersion.SecurityVersion;
+            }
+        }
+
+        public TrustVersion TrustVersion
+        {
+            get
+            {
+                return MessageSecurityTokenVersion.TrustVersion;
+            }
+        }
+
+        public SecureConversationVersion SecureConversationVersion
+        {
+            get
+            {
+                return MessageSecurityTokenVersion.SecureConversationVersion;
+            }
+        }
+
+        public SecurityTokenVersion SecurityTokenVersion
+        {
+            get
+            {
+                return MessageSecurityTokenVersion;
+            }
+        }
+
+        public abstract SecurityPolicyVersion SecurityPolicyVersion { get; }
+        public abstract BasicSecurityProfileVersion BasicSecurityProfileVersion { get; }
+        internal abstract MessageSecurityTokenVersion MessageSecurityTokenVersion { get; }
+
+        class WSSecurity11WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11MessageSecurityVersion : MessageSecurityVersion
+        {
```

Comment:
Explicit accessor needed. Should be internal. Use VS code fix to fix whole file.